src/_locales/en/messages.json

```diff
@@ -216,27 +216,27 @@
     "version": "Version",
     "welcome_0_close_button": "I'll do it later",
     "welcome_0_restore_key": "I want to restore my backup",
-    "welcome_1a1_didntfind": "Didn't find your account?",
-    "welcome_1a1_title": "Setup which account?",
+    "welcome_1a1_didntfind": "Didn't you find your account?",
+    "welcome_1a1_title": "To set up which account?",
     "welcome_1a1b_connect": "Connect",
     "welcome_1a1b_danger": "Censored network",
     "welcome_1a1b_switch": "Use existing accounts",
     "welcome_1a1b_title": "Maskbook supports these social networks",
-    "welcome_1a2_description": "Then only people you select with Maskbook can see the posted content.",
+    "welcome_1a2_description": "Then, only friends you select with Maskbook can see the posted content.",
```

Comment:
It is a good time to start reducing the mentions of "select". Manual recipient selection is optional and not mandatory. I suggest "Then, only your friends can see the content via Maskbook. (You have the option to select specific friends, too.)" for this field.